src/libraries/Pausable.sol

```diff
@@ -105,4 +179,31 @@ abstract contract Pausable is Initializable {
         PauseStorage storage $ = _getPauseStorage();
         return $._pauseFlag == PAUSED;
     }
+
+    /**
+     * @dev Transfers the ability to pause to a new account (`newPauser`).
+     */
+    function _transferPauserCapability(
+        address newPauser,
+        address owner
+    ) internal virtual onlyOwnerOrPauser(owner) {
+        PauserStorage storage $ = _getPauserStorage();
+        address oldPauser = $._pauser;
+        $._pauser = newPauser;
+        emit PauserTransferred(oldPauser, newPauser);
+    }
+
+    /**
+     * @dev Leaves the contract without a Pauser
+     * and all the capabilities afforded with that role.
+     * TODO: do we need this?
+     */
+    function renouncePauser(address owner) public virtual onlyOwnerOrPauser(owner) {
```

Comment:
Anyone can now pause the contract by specifying their own address as the pauser :(